### DISCUSSION TITLE
用其他展票测试的时候出现未知错误

### DISCUSSION THREAD

**Discussion by mtanjigx** [mikumifa/biliTickerBuy#226] [提问专区]
![345d0ed1-548d-433f-8e24-e35c986aba31](https://github.com/mikumifa/biliTickerBuy/assets/174166786/50608806-de08-439e-86f4-ab0244215594)

**Comment by mikumifa** [comment]
#131

**Reply by mtanjigx** [reply]
好的感谢老师，那对于bw的话直接按照这么运行就可以了吗

**Reply by mtanjigx** [reply]
> #131

老师，刚才用了一个需要全实名的展子测试一次多张还是显示未知错误
